docs/browser/events.md

```diff
@@ -0,0 +1,158 @@
+
+**[1]:**  Alias for [`http.url`](../../../trace/semantic_conventions/http.md)
+**[2]:** The URL fragment may be included for virtual pages
+
+`type` MUST be one of the following:
+
+| Value  | Description |
+|---|---|
+| `0` | physical_page - Initial page load within the browser and will generally also precede a PageNavigationTiming event.|
+| `1` | virtual_page - This is for Single Page Applications (SPA) where the framework provides the ability to perform client side only page "navigation", the exact definition of what a virtual page change is determined by the SPA and the framework it is using.|
+
+## PageNavigationTiming
+
+The event name MUST be`page_navigation_timing`.
+
+This event describes the timing metrics of a page navigation as provided by
+`PerformanceNavigationTiming` Performance API.
+
+The following table describes the payload fields that MUST be used to describe the details of event.
+
+| Body Field  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| name | string | URL of the page || Recommended |
+| fetchStart | long | || Recommended |
+| unloadEventStart | long | || Recommended |
+| unloadEventEnd | long | || Recommended |
+| domInteractive | long | || Recommended |
+| domContentLoadedEventStart | long | || Recommended |
+| domContentLoadedEventEnd | long | || Recommended |
+| domComplete | long | || Recommended |
+| loadEventStart | long | || Recommended |
+| loadEventEnd | long | || Recommended |
+| firstPaint | long | || Recommended |
+| firstContentfulPaint | long | || Recommended |
+
+## ResourceTiming
+
+The event name MUST be `resource_timing`.
+
+This event describes the timing metrics as provided by `PerformanceResourceTiming`
+Performance API.  These metrics are related to fetching a resource, such as
+XMLHttpRequest, Fetch, sendBeacon APIs, SVG, image or script.
+
+The following table describes the payload fields that MUST be used to describe the details of event.
+
+| Body Field  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+|name | string | URL of the requested resource || Recommended |
+|fetchStart | long | || Recommended |
+|domainLookupStart | long | || Recommended |
+|domainLookupEnd | long | || Recommended |
+|connectStart | long | || Recommended |
+|secureConnectionStart | long | || Recommended |
+|connectEnd | long | || Recommended |
+|requestStart | long | || Recommended |
+|responseStart | long | || Recommended |
+|responseEnd | long | || Recommended |
+
+## Exception
+
+The event name MUST be `browser.exception`.
+
+This event describes an error or exception that occurs in a browser application.
+
+The following table describes the payload fields that MUST be used to describe the details of event.
+
+| Body Field  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `exception.file_name` | string | Name of the file that generated the error | `foo.js` | Recommended |
```

Comment:
Assuming it followed exception log events semconv, we could expand it with existing code attributes    https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/code.md instead of adding a bunch of similar `exception*` attributes

Reply:
Exception event is being tracked separately. https://github.com/open-telemetry/semantic-conventions/pull/970